app/schemas/discogs.schemas.ts

```diff
@@ -0,0 +1,14 @@
+import z from "zod";
+
+const DiscogsResponseSchema = z.object({
```

Comment:
this is specifically for the Discog's search response, we may of have responses in the future

```suggestion
const DiscogsSearchResponseSchema = z.object({
```